DotNet/Tenant/Services/FacilityConfigurationService.cs

```diff
@@ -283,59 +282,72 @@ private void ValidateFacility(FacilityConfigModel facility)
+                            }
                         }
                     }
                 }
             }
             // validate topic is unique within Facility
 
-            IEnumerable<string> duplicates = facility.ScheduledTasks.GroupBy(i => i.KafkaTopic).Where(g => g.Count() > 1).Select(g => g.Key);
+            IEnumerable<string?> duplicates = facility.ScheduledTasks.GroupBy(i => i.KafkaTopic).Where(g => g.Count() > 1).Select(g => g.Key);
 
             string duplicatedTopics = string.Join(" ", duplicates.ToList());
 
             if (!duplicatedTopics.Equals(""))
             {
-                this._logger.LogError($"The following topics {duplicatedTopics} are duplicated for facility {facility.FacilityId} ");
+                _logger.LogError($"The following topics {duplicatedTopics} are duplicated for facility {facility.FacilityId} ");
                 throw new ApplicationException($"The following topics {duplicatedTopics} are duplicated for facility {facility.FacilityId} ");
             }
             // validate report Type within Topic
             StringBuilder schedulingErrors = new StringBuilder();
             foreach (ScheduledTaskModel facilityScheduledTask in facility.ScheduledTasks)
             {
 
-                IEnumerable<string> duplicatedReportTypes = facilityScheduledTask.ReportTypeSchedules.GroupBy(i => i.ReportType).Where(g => g.Count() > 1).Select(g => g.Key);
+                IEnumerable<string?> duplicatedReportTypes = facilityScheduledTask.ReportTypeSchedules.GroupBy(i => i.ReportType).Where(g => g.Count() > 1).Select(g => g.Key);
 
                 string duplicatedReportTypesString = string.Join(" ", duplicatedReportTypes.ToList());
 
                 if (!duplicatedReportTypesString.Equals(""))
                 {
-                    this._logger.LogError($"The following ReportTypes {duplicatedReportTypesString} are duplicated for facility {facility.FacilityId} and KafakaTopic {facilityScheduledTask.KafkaTopic}");
+                    _logger.LogError($"The following ReportTypes {duplicatedReportTypesString} are duplicated for facility {facility.FacilityId} and KafakaTopic {facilityScheduledTask.KafkaTopic}");
                     throw new ApplicationException($"The following ReportTypes {duplicatedReportTypesString} are duplicated for facility {facility.FacilityId} and KafakaTopic {facilityScheduledTask.KafkaTopic}");
                 }
+
                 int i = 1;
                 foreach (ReportTypeSchedule reportTypeSchedule in facilityScheduledTask.ReportTypeSchedules)
                 {
+                    if (reportTypeSchedule.ReportType == null)
+                    {
+                        throw new ApplicationException($"ReportType under KafkaTopic {facilityScheduledTask.KafkaTopic} must be specified.");
```

Comment:
This exception about ReportType is captured below in schedulingErrors along with others to be reported together.